lib/app.js

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var express = require('express');
+var bodyParser = require('body-parser');
+var cors = require('cors');
+
+var config = require('./config');
+var verifyOAuthToken = require('./verify');
+var logSummary = require('./logging/summary');
+var routes = require('./routes');
+
+var CORS_ORIGIN = config.get('corsOrigin');
+
+module.exports = function initApp() {
+  var app = express();
+  app.use(bodyParser.json());
+  app.use(logSummary());
+  app.use(cors({
+    origin: CORS_ORIGIN
+  }));
+
+  app.use(verifyOAuthToken());
```

Comment:
Should we use `app.set('x-powered-by', false);`? See http://expressjs.com/4x/api.html#app.set